gmusicapi/clients/musicmanager.py

```diff
@@ -1,17 +1,19 @@
 from __future__ import print_function, division, absolute_import, unicode_literals
-from future import standard_library
-standard_library.install_aliases()
+from future.utils import PY3
 from past.builtins import basestring
 from builtins import *  # noqa
+
 import os
 from socket import gethostname
 import time
-import urllib.request
-import urllib.parse
-import urllib.error
 from uuid import getnode as getmac
 import webbrowser
 
+if PY3:
```

Comment:
`six` is already required by the base package (there's a comment "raise_from on Python 3" in setup.py) as well as numerous other libraries required by gmusicapi so it won't actually add another compat library. :)